src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -82,6 +83,12 @@ public override int Order
         public string Route { get; set; }
 
         /// <summary>
+        /// The http method to use.
+        /// </summary>
+        /// <remarks>Passed through to the generated HTML in all cases.</remarks>
+        public string Method { get; set; }
```

Comment:
Lets add an `EditorBrowsable.Never` decorator to this. No value in showing consumers we're binding their `Method` attribute.